Detectors/TRD/macros/CreateCCDBLocalGainFactor.C

```diff
@@ -0,0 +1,43 @@
+#include "TRDCalibration/PadCalibCCDBBuilder.h"
+
+#include <ctime>
+#include <sstream>
+#include <string>
+
+using namespace o2::trd;
+
+void CreateCCDBLocalGainFactor(TString sOpenFile)
+{
+  PadCalibCCDBBuilder* krCalibToCCDB = new PadCalibCCDBBuilder();
+
+  TFile* file = TFile::Open(sOpenFile);
+  TTree* tree = (TTree*)file->Get("nt_Krypton");
```

Comment:
The file will stay open after the function call and the tree will persist as well.
Either drop both at the end or use:
```
std::unique_ptr<TFile> file( TFile::Open(sOpenFile) );
auto tree = file->Get<TTree>("nt_Krypton");
```

Reply:
deleted after the loop